stream-chat-android-ui-common/api/stream-chat-android-ui-common.api

```diff
@@ -180,6 +180,25 @@ public final class com/getstream/sdk/chat/model/AttachmentMetaData {
 	public fun toString ()Ljava/lang/String;
 }
 
+public final class com/getstream/sdk/chat/model/GiphyInfo {
```

Comment:
Do we need to expose anything new in this module to clients? I feel like there should be no changes in this `.api` file on this PR, unless I'm missing something.

Reply:
No need to make `GiphyInfo`. Let me check for updates in the API

Reply:
🤔  I think it makes sense to change the API to remove `streamUiMediaAttachmentGiphyHeight` as now giphys are auto sizeable... or we can keep it so users can choose to have it the same size always.

Reply:
I think it's fine to keep it as an optional attribute. If we do that, let's make sure we document it in the attrs file properly.